### PR TITLE
Fixes #14243

### LINKED ISSUE #14243: [23:05:35] Runtime in unsorted.dm, line 1610: Unknown encoding received from client: "_autodetect". Please report this as a bug.
```

proc name: stack trace (/proc/stack_trace)
usr: JohnExplode (johnexplode) (/mob/new_player)
usr.loc: The floor (8, 81, 2) (/turf/unsimulated/floor)
src: null
call stack:
stack trace("Unknown encoding received from...")
/datum/chatOutput (/datum/chatOutput): Topic("_src_=chat&proc=encoding&encod...", /list (/list))
JohnExplode (/client): Topic("_src_=chat&proc=encoding&encod...", /list (/list), null)
JohnExplode (/client): Topic("_src_=chat&proc=encoding&encod...", /list (/list))

```

🤷‍♂️

### PR DESCRIPTION
**Pull request by PJB3005**
Fixes #14243

### CODE REVIEW

File: goon/code/datums/browserOutput.dm [hunk 1]
```diff
@@ -98,12 +98,20 @@ For the main html chat area
...
 
 		if("encoding")
+			var/encoding = href_list["encoding"]
 			var/static/regex/RE = regex("windows-(874|125\[0-8])")
-			if (RE.Find(href_list["encoding"]))
-				world.log << "ENCODING RECEIVED: [href_list["encoding"]]"
+			if (RE.Find(encoding))
 				owner.encoding = RE.group[1]
+
+			else if (encoding == "gb2312")
```
Comment (clusterfack) [inline goon/code/datums/browserOutput.dm]: what the fuuuuuuuuuuck

Comment (PJB3005) [reply goon/code/datums/browserOutput.dm:106]: Chinese people.

Comment (gbasood) [reply goon/code/datums/browserOutput.dm:106]: maybe add a comment or something so it isn't an arcane thing people have to google every time they want to know what encoding it is

Comment (Probe1) [reply goon/code/datums/browserOutput.dm:106]: Add the comment in a separate PR if you want this PRs been rotting on the vine.  ☹️

**Pull request merged**